guides/index.md

```diff
@@ -11,37 +11,43 @@ for information on the latest releases and new features.
-By default, Ember apps created through Ember CLI include the `ember-auto-import` dependency, which enables importing npm packages directly. For example, if you want to use the `highcharts` library in your application,
-you can install and import it in your application without any other configuration. Be sure to visit the [ember-auto-import](https://github.com/ef4/ember-auto-import) documentation for more advanced usage!
+Since `v3.x`, Ember apps include the `ember-auto-import` dependency, which enables importing npm packages directly. For example, if you want to use `highcharts` in your application,
+you can install and import it without any other configuration. Be sure to visit the [ember-auto-import](https://github.com/ef4/ember-auto-import) documentation for more advanced usage!
 
 ## Why do we need a CLI?
 
-The Ember CLI is like a dependency packager, test runner, optimizer, and local server all rolled into one. Since all the features were built to work together, common tasks (such as upgrading the app version or deploying) can be automated with production-ready, open source plugins. The CLI is backwards-compatible with older Ember apps and maintains a six-week release schedule.
+The Ember CLI's job is to make your work easier.
 
-The CLI's job is to make your work easier.
-It was built with the philosophy that a developer should be able to focus on building great apps, not re-engineering how to fit all the pieces together at each stage of an app's lifecycle. The result is that apps are more maintainable and approachable, since there are established architectural patterns across individuals, teams, and companies.
+The CLI is like a dependency packager, test runner, optimizer, and local server—all rolled into one. Since all features were built to work together, common tasks (e.g. upgrading the app version and deploying the app) can be automated with production-ready, open source plugins. The CLI is backwards-compatible with older Ember apps and maintains a 6-week release schedule.
 
-However, if you want to make your own adjustments like using a different testing library, you still can.
+The CLI was also built with the idea that a developer should be able to focus on building great apps, not re-engineering how to fit pieces together throughout an app's lifecycle. Apps become maintainable and approachable, since there are established architectural patterns across individuals, teams, and companies.
+However, if you want to make your own adjustments, such as using a different testing library, you still can.
 
 ## Getting help
 
-Do you have questions? Run into an issue or a bug? Get support from the community. A list of chat rooms, forums, and more are available [here](https://www.emberjs.com/community/).
+Did you have questions? Run into an issue or a bug? Get support from the community. A list of chat rooms, forums, and more is available on the [Community](https://www.emberjs.com/community/) page.
```

Comment:
I changed "are" to "is" and provided a descriptive information for the link ("here" is, in general, discouraged, I believe).